apps/wing/src/cli.ts

```diff
@@ -1,8 +1,9 @@
-import { Command, Option } from "commander";
+import { Argument, Command, Option } from "commander";
 import { satisfies } from "compare-versions";
 
 import { optionallyDisplayDisclaimer } from "./analytics/disclaimer";
 import { exportAnalytics } from "./analytics/export";
+import { projectTemplateNames } from "./commands/init";
```

Comment:
Could `projectTemplateNames` be moved to another file? The cost for eagerly loading the imports from `init` is pretty high, and this function doesn't need them anyways.

Reply:
I'll try this